tests/unit_tests/test_server_client.py

```diff
@@ -1467,3 +1473,75 @@ async def test_batch_returns_run_id(app: FastAPI) -> None:
         assert len(run_ids) == 2
         for run_id in run_ids:
             assert _is_valid_uuid(run_id)
+
+
+@pytest.mark.asyncio
+async def test_feedback_succeeds_when_langsmith_enabled() -> None:
+    """Test the server directly via HTTP requests."""
+
+    with patch("langserve.server.ls_client") as mocked_ls_client_package:
+        mocked_client = MagicMock(return_value=None)
+        mocked_ls_client_package.Client.return_value = mocked_client
+
+        mocked_client.create_feedback.return_value = ls_schemas.Feedback(
+            id="5484c6b3-5a1a-4a87-b2c7-2e39e7a7e4ac",
+            created_at=datetime.datetime(1994, 9, 19, 9, 19),
+            modified_at=datetime.datetime(1994, 9, 19, 9, 19),
+            run_id="f47ac10b-58cc-4372-a567-0e02b2c3d479",
+            key="silliness",
+            score=1000,
+        )
+
+        local_app = FastAPI()
+        add_routes(
+            local_app,
+            RunnableLambda(lambda foo: "hello"),
+        )
+
+        async with get_async_test_client(
+            local_app, raise_app_exceptions=True
+        ) as async_client:
+            response = await async_client.post(
+                "/feedback",
+                json={
+                    "run_id": "f47ac10b-58cc-4372-a567-0e02b2c3d479",
+                    "key": "silliness",
+                    "score": 1000,
+                },
+            )
+
+            expected_response_json = {
+                "run_id": "f47ac10b-58cc-4372-a567-0e02b2c3d479",
+                "key": "silliness",
+                "score": 1000,
+                "created_at": datetime.datetime(1994, 9, 19, 9, 19).strftime(
+                    "%Y-%m-%dT%H:%M:%S"
+                ),
+                "modified_at": datetime.datetime(1994, 9, 19, 9, 19).strftime(
+                    "%Y-%m-%dT%H:%M:%S"
+                ),
+                "comment": None,
```

Comment:
@eyurtsev do we have a standard on how we deal with empty fields? Do we expose them set to None, or hide them? 

I know whichever format we stick to ends up having downstream affects on things like partial update APIs, so I figure it makes sense to follow whatever our standard is. Leaving this for now (which isn't optimal), but will change it based on your guidance